repose-aggregator/components/repose-lint/pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.openrepose</groupId>
+        <artifactId>components-support</artifactId>
+        <version>7.3.1.0-SNAPSHOT</version>
+    </parent>
+
+    <name>Repose Components - Repose Lint</name>
+    <artifactId>repose-lint</artifactId>
+    <packaging>jar</packaging>
+
+    <description>
+        A command-line utility which flags suspicious usage of Repose.
+    </description>
+
+    <properties>
+        <sonar.jacoco.itReportPath>${project.basedir}/../../target/jacoco-it.exec</sonar.jacoco.itReportPath>
+    </properties>
+
+    <dependencies>
+        <!-- Scala language support -->
+        <dependency>
```

Comment:
It looks like the dependency is managed, but not declared as a hard dependency at the "profile-support" level.